controls/bsi_sys_1_3_rhcos4.yml

```diff
@@ -0,0 +1,429 @@
+title: 'SYS.1.3 Linux Server (RHCOS)'
+id: bsi_sys_1_3_rhcos4
+version: '1.0'
+source: https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Grundschutz/International/bsi_it_gs_comp_2022.pdf
+levels:
+  - id: basic
+  - id: standard
+    inherits_from:
+      - basic
+  - id: elevated
+    inherits_from:
+      - standard
+
+reference_type: bsi
+product: rhcos4
+
+controls:
+  - id: SYS.1.3.A1
+    title: ELIMINATED
+    levels:
+      - basic
+    description: >-
+      This requirement has been eliminated.
+    notes: >-
+      This requirement has been eliminated.
+    status: not applicable
+
+  - id: SYS.1.3.A2
+    title: Careful Allocation of IDs
+    levels:
+      - basic
+    description: >-
+      (1) Each login name, each user ID (UID) and each group ID (GID) MUST ONLY be used once.
+      (2) Every user MUST be a member of at least one group. (3) Every GID mentioned in the /etc/passwd
+      file MUST be defined in the /etc/group file. (4) Every group SHOULD only contain the users that
+      are absolutely necessary. (5) In networked systems, care MUST also be taken to ensure that user
+      and group names (UIDs and GIDs) are assigned consistently in the system network if there is a
+      possibility that the same UIDs or GIDs could be assigned to different user or group names on
+      the systems during cross-system access.
+    notes: >-
+      Section 2: System accounts do not always have a group
+      Section 4: is a manual control
+      Section 5: this cant be checked on a per system base, and therefore is an organizational control
+    status: partial
+    rules:
+      # Section 1
+      - account_unique_id
+      - account_unique_name
+      - group_unique_id
+      - group_unique_name
+      # Section 2
+      # this could be automated
+      # Section 3
+      - gid_passwd_group_same
+
+  - id: SYS.1.3.A3
+    title: No Automatic Integration of Removable Drives
+    levels:
+      - basic
+    description: >-
+      (1) Removable media such as USB pen drives or CDs/DVDs MUST NOT be integrated automatically.
+    notes: >-
+      https://access.redhat.com/solutions/18978
+    status: automated
+    rules:
+      # USB
+      - grub2_nousb_argument
+      - coreos_nousb_kernel_argument
+      - bios_disable_usb_boot
+      - kernel_module_usb-storage_disabled
+      # Automount
+      - service_autofs_disabled
+
+  - id: SYS.1.3.A4
+    title: Protection from Exploitation of Vulnerabilities in Applications
+    levels:
+      - basic
+    description: >-
+      (1) ASLR and DEP/NX MUST be activated in the kernel and used by applications to make it harder
+      to exploit vulnerabilities in applications. (2) Security functions of the kernel and of the standard
+      libraries (such as heap and stack protection) MUST NOT be disabled.
+    notes: >-
+      This should be the default on all modern platforms
+      Section 2: organizational requirement towards the admin
+    status: automated
+    rules:
```

Comment:
Would it make sense to include?
- `sysctl_kernel_randomize_va_space`
- `coreos_pti_kernel_argument`
- `coreos_page_poison_kernel_argument`
- `coreos_slub_debug_kernel_argument`

Reply:
it does, they will be added.